function.rb

```diff
@@ -0,0 +1,9 @@
+
+def function(n)
+  if n == 0
+    1
+  else
+    n * function(n-1)
```

Comment:
an explicit return here and in line 4 would add clarity to what's being returned.